packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -550,8 +550,37 @@ This is only applicable when previewing Android releases.''',
     final logs = await open.newApplication(path: appDirectory.path);
     final completer = Completer<void>();
 
+    // Filter out extra noise from the logs.
+    final logFilters = [
+      RegExp(r'\[com\.apple.*\]'),
```

Comment:
I'm surprised macos doesn't have an easy way to do the opposite and just filter for the flutter stuff 🤷  But I guess that would require running a separate process since this is just a direct dump from stdout/stderr.